Tools.SSOWithKombitSts/SamlMetadataTemplate.xml

```diff
@@ -1,28 +1,61 @@
+            </X509Data>
+        </KeyInfo>
+    </Signature>
+    -->
+    <md:SPSSODescriptor protocolSupportEnumeration="urn:oasis:names:tc:SAML:2.0:protocol" AuthnRequestsSigned="true" WantAssertionsSigned="true">
         <md:KeyDescriptor use="signing">
-            <ds:KeyInfo xmlns:ds="http://www.w3.org/2000/09/xmldsig#">
-                <ds:X509Data>
-                    <ds:X509Certificate></ds:X509Certificate>
-                </ds:X509Data>
-            </ds:KeyInfo>
+            <KeyInfo xmlns="http://www.w3.org/2000/09/xmldsig#">
+                <X509Data>
+                    <X509Certificate></X509Certificate>
+                </X509Data>
+            </KeyInfo>
         </md:KeyDescriptor>
         <md:KeyDescriptor use="encryption">
-            <ds:KeyInfo xmlns:ds="http://www.w3.org/2000/09/xmldsig#">
-                <ds:X509Data>
-                    <ds:X509Certificate></ds:X509Certificate>
-                </ds:X509Data>
-            </ds:KeyInfo>
+            <KeyInfo xmlns="http://www.w3.org/2000/09/xmldsig#">
+                <X509Data>
+                    <X509Certificate></X509Certificate>
+                </X509Data>
+            </KeyInfo>
         </md:KeyDescriptor>
-        <md:SingleLogoutService Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST" Location="https://demo-brugervendtsystem.kombit.dk/test/saml/SingleLogout"/>
-        <md:SingleLogoutService Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect" Location="https://demo-brugervendtsystem.kombit.dk/test/saml/SingleLogout"/>
+        <md:SingleLogoutService Binding="urn:oasis:names:tc:SAML:2.0:bindings:HTTP-POST" Location="https://localhost:44300/Logout.ashx" ResponseLocation="https://localhost:44300/Logout.ashx" />
```

Comment:
Her står localhost url'er... hvis det er dette dokument der danner udgangspunkt for vores metadata på sts, så skal det vel udvides med de urler som vi har tænkt os at bruge dvs. med kitos-integration.strongminds.dk, kitostest.miracle.dk og kitos.dk